package.json

```diff
@@ -48,7 +48,7 @@
     "pub": "sh build/release.sh",
     "postinstall": "opencollective-postinstall"
   },
-  "husky": {
+  "husky-OFF": {
```

Comment:
temporary. if we accept any changes, they should all be formatted.

Actually we should just go ahead and format the whole code base all in one PR and get it out of the way if we're going to have the eslint/prettier configs in the repo.